README.md

```diff
@@ -24,13 +37,16 @@ create-scxa-analytics-collection.sh
 ```
 
 ## Enable BioSolr
-`scxa-analytics-v5` makes use of the [BioSolr plugin](https://github.com/ebi-gene-expression-group/BioSolr) to perform ontology expansion on document indexing. In order to enable BioSolr, there are 2 options:
+
+`scxa-analytics-v5` makes use of the [BioSolr plugin](https://github.com/ebi-gene-expression-group/BioSolr) to perform ontology expansion on document indexing. In order to enable BioSolr, there are 3 options:
```

Comment:
`scxa-analytics-v7`? :wink:

Reply:
Yeah, probably at least v6. We didn't change the schema here (to move to v7), but we could as well do if you prefer.